### PR TITLE
Update csrankings.csv

### PR DESCRIPTION
**Pull request by collider123**
add some Cleveland State University faculty

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
1. Please remove spaces after commas in every entry.
2. Please add _every_ tenure-track faculty member listed here: https://engineering.csuohio.edu/eecs/electrical-computer-engineering-faculty

> If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

**Comment by collider123** [comment]
All CS tenure-track faculty listed here https://engineering.csuohio.edu/eecs/computer-science-faculty and here https://engineering.csuohio.edu/eecs/electrical-computer-engineering-faculty have been added.

Thanks.

File: csrankings.csv [hunk 1]
```diff
@@ -2936,6 +2937,7 @@ Changwoo Min,Virginia Tech,https://sites.google.com/site/multics69,6VDjaN4AAAAJ
 Changxi Zheng,Columbia University,http://www.cs.columbia.edu/~cxz,-0rEuLgAAAAJ
 Changyou Chen,University at Buffalo,https://cse.buffalo.edu/~changyou,LtEcKBcAAAAJ
 Chanik Park,POSTECH,http://cpmi.postech.ac.kr/prof-chanik-park,TsP0SGcAAAAJ
+Chansu Yu,https://academic.csuohio.edu/yuc/,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv:2940]: Missing affiliation

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (collider123) [reply csrankings.csv:2940]: missing affiliation added. Thank you

**Pull request merged**